sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -515,6 +515,35 @@ public void StartRecognizeContentWithNoSupporttedLanguage()
             Assert.AreEqual("NotSupportedLanguage", ex.ErrorCode);
         }
 
+        [RecordedTest]
+        public async Task StartRecognizeContentWithReadingOrder()
+        {
+            var client = CreateFormRecognizerClient();
+            var uri = FormRecognizerTestEnvironment.CreateUri(TestFile.Form1);
+            RecognizeContentOperation basicOrderOperation, naturalOrderOperation;
+
+            basicOrderOperation = await client.StartRecognizeContentFromUriAsync(uri, new RecognizeContentOptions() { ReadingOrder = ReadingOrder.Basic });
+            naturalOrderOperation = await client.StartRecognizeContentFromUriAsync(uri, new RecognizeContentOptions() { ReadingOrder = ReadingOrder.Natural });
+
+            await basicOrderOperation.WaitForCompletionAsync(PollingInterval);
+            Assert.IsTrue(basicOrderOperation.HasValue);
+
+            await naturalOrderOperation.WaitForCompletionAsync(PollingInterval);
+            Assert.IsTrue(naturalOrderOperation.HasValue);
+
+            var basicOrderFormPage = basicOrderOperation.Value.Single();
+            var naturalOrderFormPage = naturalOrderOperation.Value.Single();
+
+            ValidateFormPage(basicOrderFormPage, includeFieldElements: true, expectedPageNumber: 1);
+            ValidateFormPage(naturalOrderFormPage, includeFieldElements: true, expectedPageNumber: 1);
+
+            var basicOrderLines = basicOrderFormPage.Lines.Select(f => f.Text);
+            var naturalOrderLines = naturalOrderFormPage.Lines.Select(f => f.Text);
+
+            CollectionAssert.AreEquivalent(basicOrderLines, naturalOrderLines);
+            CollectionAssert.AreNotEqual(basicOrderLines, naturalOrderLines);
```

Comment:
I think it's not worth testing whether the returned lines in fact follow the `Basic` or the `Natural` ordering algorithm (checking bounding boxes, checking proximity of related text lines, etc.). To test the SDK, simply checking that changing the specified order updated the ordering behavior in the service seems enough.